README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
 * [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Git] (https://github.com/git-for-windows/git/releases) (Select "use Git from the Windows Command Prompt" on Adjusting your PATH environement Screen")
```

Comment:
This is Github for Windows and works only on Windows. https://git-scm.com/downloads would be the right place for actually downloading Git VCS.